### PR TITLE
Add example with mixed cell types

### PR DESCRIPTION
**Pull request by dcaunt**
## Pull request checklist
- [x] All tests pass. Demo project builds and runs.
- [x] I have resolved any merge conflicts.
- [x] I have followed the [coding style](https://github.com/jessesquires/HowToContribute#style-guidelines), and reviewed the [contributing guidelines](https://github.com/jessesquires/HowToContribute).
## What's in this pull request?

An example implementing the approach discussed in #57

Let me know if you'd like to revise the example in any way or if there are any issues.

### CODE REVIEW

File: Example/Sources/FancyCollectionViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+//  Released under an MIT license: http://opensource.org/licenses/MIT
+//
+
+import UIKit
+
+class FancyCollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet weak var label: UILabel!
+
+    var testing = false
```
Comment (jessesquires) [inline Example/Sources/FancyCollectionViewCell.swift]: remove this? 😄

**Comment by jessesquires** [comment]
Thanks so much @dcaunt ! This is awesome ✨ 🎉

**Comment by jessesquires** [comment]
1 small comment. Looks like some left over debugging code?

**Comment by jessesquires** [comment]
After we merge this, would you be interested helping with the following:
1. merging `develop` into `swift2.3`, migrate, submit PR
2. then merging `swift2.3` into `swift3.0`, migrate, submit PR

No pressure if you don't have time, or don't want to! 😊 I can take care of this. Just wanted to ask.

File: Example/Sources/FancyCollectionViewCell.swift (hunk 1 above)
Comment (dcaunt) [inline Example/Sources/FancyCollectionViewCell.swift]: Oh haha, will do

**Comment by dcaunt** [comment]
Yes, I can migrate this PR to the other Swift versions, it's not much work :)

**Comment by jessesquires** [comment]
Thanks @dcaunt ! 🎉

**Pull request merged**